### ISSUE TITLE
problem with start.sh

### ISSUE THREAD

**Issue by Zengdun-cs** [fabienwang/passport_cn#1]
I got this error when I run start.sh. Could you help me with this? thx a lot.

Traceback (most recent call last):
  File "/home/zed/passport_cn/holyscrap.py", line 41, in <module>
    driver = webdriver.Remote(
  File "/home/zed/passport_cn/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 157, in __init__
    self.start_session(capabilities, browser_profile)
  File "/home/zed/passport_cn/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 252, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "/home/zed/passport_cn/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/zed/passport_cn/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: Unable to create new service: GeckoDriverService
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'greg', ip: '47.101.132.200', os.name: 'Linux', os.arch: 'amd64', os.version: '5.12.15-arch1-1', java.version: '11.0.11'
Driver info: driver.version: unknown

**Comment by fabienwang** [comment]
Hello, you are missing the selenium driver, `sudo pacman -S geckodriver`.
You can also use chrome if you prefer: `sudo pacman -S chromedriver` and adjust your `config.py` first line :)

**Comment by Zengdun-cs** [comment]
I installed driver as you said. It is still not working (may be something wrong with PATH that I can‘t solve it).
So I changed driver initialization, let it run in a single python script. 
Like this,

chrome_driver = "./chromedriver"
driver = webdriver.Chrome(executable_path=chrome_driver)

It worked. But new error comes.

selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[6]/div[3]/div/button"}
  (Session info: chrome=91.0.4472.164)

This appoinments is not open yet. Is that why this Button can not be founded？
